caffe2/CMakeLists.txt

```diff
@@ -1191,6 +1191,7 @@ if (BUILD_TEST)
     add_executable(${test_name} "${test_src}")
     target_link_libraries(${test_name} ${Caffe2_MAIN_LIBS} gtest_main)
     target_include_directories(${test_name} PRIVATE $<INSTALL_INTERFACE:include>)
+    target_include_directories(${test_name} PRIVATE $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/include>) 
```

Comment:
@resistor @ZolotukhinM @bertmaher @nickgg I'll send a PR to PT trunk for this as well